Interface/AddOns/oUF_Neav/core.lua

```diff
@@ -211,7 +261,7 @@ local function PlayerToVehicleTexture(self, event, unit)
     self.RaidIcon:SetPoint('CENTER', self.Portrait, 'TOP', 0, -5)
     self.T[1]:SetPoint('BOTTOM', self.Name, 'TOP', 0, 8)
 
-    securecall('PlayerFrame_ShowVehicleTexture')
+    HideAltResources(self)
```

Comment:
```
function PlayerFrame_ShowVehicleTexture()
	PlayerFrameVehicleTexture:Show();
	
	local _, class = UnitClass("player");	
	if ( PlayerFrame.classPowerBar ) then
		PlayerFrame.classPowerBar:Hide();
	elseif ( class == "SHAMAN" ) then
		TotemFrame:Hide();
	elseif ( class == "DRUID" ) then
		EclipseBarFrame:Hide();
	elseif ( class == "DEATHKNIGHT" ) then
		RuneFrame:Hide();
	elseif ( class == "PRIEST" ) then
		PriestBarFrame:Hide();
	end
end
```

Is the entire function and we don't need the `PlayerFrameVehicleTexture:Show()` call and the rest of it doesn't cover all of what we need.